packages/core/mesh/messaging/src/signal-manager/websocket-signal-manager.ts

```diff
@@ -24,13 +26,18 @@ const WSS_SIGNAL_SERVER_REBOOT_DELAY = 3_000;
  */
 export class WebsocketSignalManager implements SignalManager {
   private readonly _servers = new Map<string, SignalClientMethods>();
+  private readonly _monitor = new WebsocketSignalManagerMonitor();
+
+  /**
+   * Used to avoid logging failed server restarts more than once until the server actually recovers.
+   */
+  private readonly _failedServersBitfield: Uint8Array;
```

Comment:
Does the client do any backoff on failure or are we just suppressing log messages here?

Reply:
client has a static 3s wait delay, but the field is used only for log message suppression